.github/workflows/build-all.yml

```
@@ -167,9 +170,21 @@ jobs:
         security import $CERTIFICATE_PATH -P "$MACOS_BUILD_CERT_P12_PASSWORD" -A -t cert -f pkcs12 -k $KEYCHAIN_PATH
         security list-keychain -d user -s $KEYCHAIN_PATH
 
-        codesign -s 'Fossa, Inc.' release/fossa
-        codesign -s 'Fossa, Inc.' release/pathfinder
-        codesign -s 'Fossa, Inc.' release/diagnose
+        # '--options runtime' enables the hardened runtime: https://developer.apple.com/documentation/security/hardened_runtime
+        # The hardened runtime is required for notarization.
+        codesign --options runtime -s 'Fossa, Inc.' release/fossa
+        codesign --options runtime -s 'Fossa, Inc.' release/pathfinder
+        codesign --options runtime -s 'Fossa, Inc.' release/diagnose
```

The company name is FOSSA (in all caps).
```suggestion
        codesign --options runtime -s 'FOSSA, Inc.' release/fossa
        codesign --options runtime -s 'FOSSA, Inc.' release/pathfinder
        codesign --options runtime -s 'FOSSA, Inc.' release/diagnose
```